docs/source/images/alignment_plots/same_inits_alignment.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
do you think you can put it all in one notebook?

Reply:
Yes, I'll merge these in the next PR.